<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeedTestPage</class>
 <widget class="QWidget" name="SpeedTestPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(51,153,255);
}

QLabel
{
background-color:rgb(51,153,255);
border:0px solid white;
font:20px &quot;SimHei&quot;;
color:white;
}

QPushButton
{
outline:none;
font:25px, &quot;SimHei&quot;;
}</string>
  </property>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>列车速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSpeed">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelLevel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>列车级位</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTestSpeed">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTestLevel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>测试级位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>测试速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>末级位</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOriSpeed">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>初速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEndSpeed">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>测试距离</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTestDistance">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTest1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">outline:none;
border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string>加速度1测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTest2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">outline:none;
border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string>加速度2测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTestMinus">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">outline:none;
border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string>减速度测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">outline:none;
border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>511</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:16px;
color:red;</string>
   </property>
   <property name="text">
    <string>1.请在列车速度为0时设定加速度测试模式
2.加速度测试设定后，牵引指令有效时开始计算
3.加速度测试时，请在列车速度小于10kmph内将手柄置于所需级位
4.超过10kmph后级位变化或达到目标速度，加速度测试自动完成
5.减速度测试设定后，制动指令有效时测试开始，列车停止时测试完成</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
